tests/time_tests/test_runner/.automation/desktop_test_config.yml

```diff
@@ -474,3 +474,17 @@
     name: icnet-camvid-ava-sparse-60-0001
     precision: FP16-INT8
     framework: tf
+- device:
+    name: CPU
+  model:
+    path: ${MODELS_PATH}/ModelA/INT8/ModelA_INT8.xml
+    name: ModelA_INT8
+    precision: INT8
+    framework: unknown
+- device:
+    name: CPU
+  model:
+    path: ${MODELS_PATH}/ModelE/INT8/ModelE_INT8.xml
+    name: ModelA_INT8
+    precision: INT8
+    framework: unknown
```

Comment:
original framework is TF - it may be found in .xml file in `<cli_parameters>` tag

Reply:
Oh, thanks for information :)